### ISSUE TITLE
Support Azure Container Registry

### ISSUE THREAD

**Issue by lizrice** [aquasecurity/starboard#278]
We have documentation for using Starboard to scan images in Amazon's ECR, we should work out what the equivalent is for Azure

**Comment by krol3** [comment]
I'd like to help :)

**Comment by NissesSenap** [comment]
@krol3 have you had any time to look in to this?

**Comment by NissesSenap** [comment]
Waiting for https://github.com/aquasecurity/fanal/pull/371 great work @cgroschupp for fixing the PR:s in all these repos!

**Comment by krol3** [comment]
@NissesSenap the PR was merged.

**Comment by NissesSenap** [comment]
Yeah and now it was also released in trivy https://github.com/aquasecurity/trivy/releases/tag/v0.23.0
Now I know what I will focus during the rest of the week :)

**Comment by NissesSenap** [comment]
@danielpacak i guess we can close this issue thanks to @cgroschupp awesome job.
